echopype/visualize/api.py

```diff
@@ -3,13 +3,13 @@
 
 import xarray as xr
 
-from .plot import _plot_echogram, FacetGrid, QuadMesh, T
+from .plot import _plot_echogram, FacetGrid, QuadMesh
 from ..echodata import EchoData
 
 
 def create_echogram(
     data: Union[EchoData, xr.Dataset],
-    frequency: Union[int, float, List[T], None] = None,
+    channel: Union[str, List[str], None] = None,
```

Comment:
Are we changing frequency input to channel now? I thought for plotting we're sticking to these for ease of use? Btw, I have a PR that deals with all of this: https://github.com/OSOceanAcoustics/echopype/pull/660. Seem like your changes will clash with mine.

Reply:
From our discussion today, it looks like we will handle this discrepancy in your PR and merge this as it is right now.